### PR TITLE
Mutation to remove 2FA from account login

### PR DESCRIPTION
**Pull request by sbinlondon**
Adds `removeTwoFactorAuthTokenFromIndividual` to enable https://github.com/opencollective/opencollective-frontend/pull/5498

### CODE REVIEW

**Review by kewitz — APPROVED** [review]
Well done, the code is looking super clean and straightforward.

File: server/graphql/v2/mutation/AccountMutations.ts [hunk 1]
```diff
@@ -161,6 +162,51 @@ const accountMutations = {
       return account;
     },
   },
+  removeTwoFactorAuthTokenFromIndividual: {
+    type: new GraphQLNonNull(Individual),
+    description: 'Add 2FA to the Account if it does not have it',
+    args: {
+      account: {
+        type: new GraphQLNonNull(AccountReferenceInput),
+        description: 'Account that will have 2FA added to it',
```
Comment (znarf) [inline server/graphql/v2/mutation/AccountMutations.ts:171]: @sbinlondon Description is wrong.

File: server/graphql/v2/mutation/AccountMutations.ts [hunk 2]
```diff
@@ -161,6 +162,51 @@ const accountMutations = {
       return account;
     },
   },
+  removeTwoFactorAuthTokenFromIndividual: {
+    type: new GraphQLNonNull(Individual),
+    description: 'Add 2FA to the Account if it does not have it',
```
Comment (znarf) [inline server/graphql/v2/mutation/AccountMutations.ts:167]: @sbinlondon also wrong description

Comment (sbinlondon) [reply server/graphql/v2/mutation/AccountMutations.ts:167]: Ok have popped it onto this PR https://github.com/opencollective/opencollective-api/pull/4946

**Pull request merged**